### PR TITLE
MINOR: Remove note in docs that is no longer required

### PR DESCRIPTION
**Pull request by rajinisivaram**

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@ijuma Can you take a look? I wasn't sure if the script does this or it needed to be done before running the release script. Thank you!

**Comment by asfgit** [comment]
FAILURE
 No test results found.
--none--

**Review by ijuma — APPROVED** [review]
LGTM, thanks.

**Comment by ijuma** [comment]
On second thought, we probably don't want to bump the version until the release vote passes.

File: docs/streams/tutorial.html [hunk 1]
```diff
@@ -62,7 +62,7 @@
     </pre>
 
     <p>
-        <strong>Important:</strong> You must manually update the setting of <code>&lt;kafka.version&gt;</code> in the generated <code>pom.xml</code> file from <code>0.11.0.1-SNAPSHOT</code> to <code>0.11.0.1</code>.
+        <strong>Important:</strong> You must manually update the setting of <code>&lt;kafka.version&gt;</code> in the generated <code>pom.xml</code> file from <code>{{fullDotVersion}}-SNAPSHOT</code> to <code>{{fullDotVersion}}</code>.
         <strong>Note:</strong> in the next release the above step will not be required.
```
Comment (ijuma) [inline docs/streams/tutorial.html]: @guozhangwang is this step still needed in 0.11.0.2?

**Comment by asfgit** [comment]
FAILURE
 No test results found.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 7149 tests run, 5 skipped, 0 failed.
--none--

File: docs/streams/tutorial.html (hunk 1 above)
Comment (guozhangwang) [reply docs/streams/tutorial.html]: As long as we now follow the release process instructions when releasing to remove the `-SNAPSHOT` suffix, it should be good. Previously in 0.11.0.1 we forgot to remove the suffix when uploading and hence added this.

We can remove this line.

**Comment by asfgit** [comment]
SUCCESS 
 7149 tests run, 5 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
FAILURE
 No test results found.
--none--

File: docs/streams/tutorial.html (hunk 1 above)
Comment (rajinisivaram) [reply docs/streams/tutorial.html]: @guozhangwang Thank you, I have removed the line. Can you review please? Thanks.

**Comment by guozhangwang** [comment]
LGTM.

**Comment by rajinisivaram** [comment]
@guozhangwang @ijuma Thanks for the review, merging to 0.11.0

**Comment by asfgit** [comment]
SUCCESS 
 7149 tests run, 5 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
FAILURE
 7149 tests run, 5 skipped, 1 failed.
--none--

**Comment by ijuma** [comment]
@rajinisivaram can you please close the PR?

**Comment by rajinisivaram** [comment]
Merged to 0.11.0

**Pull request closed without merging**